### PR TITLE
Support multiple test results per test file run.

### PR DESCRIPTION
**Pull request by doxavore**
Before this PR, we would provide Jest with a file from the manifest and get back a single test result.

After this PR, we provide Jest with a file from the manifest and get back any number of test results.

### CODE REVIEW

File: e2e/__tests__/abq.test.ts [hunk 1]
```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```
Comment (ayazhafiz) [inline e2e/__tests__/abq.test.ts]: I'm not sure what exactly we should do with the licensing headers for novel files we introduce for ABQ reasons. I don't think it's terribly important right now but maybe we want both the Facebook and an RWX attribution header? @dan-manges @TAGraves

Comment (dan-manges) [reply e2e/__tests__/abq.test.ts]: Yeah, the standard way to do this is to add an attribution line. It's not a problem if we miss one of these though as long as we're still releasing under the MIT license.

```
* Copyright (c) ReadWriteExecute, Inc. All Rights Reserved.
* Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment (doxavore) [reply e2e/__tests__/abq.test.ts]: I've added these to the new ABQ-specific files. I've not gone through to modify the header on files we may have touched that exist upstream.

Comment (dan-manges) [reply e2e/__tests__/abq.test.ts]: :+1: that's fine

**Review by kylekthompson — DISMISSED** [review]

**Review by kylekthompson — APPROVED** [review]

**Pull request merged**